### DISCUSSION TITLE
change umbraco admin url, Plumber stopped working

### DISCUSSION THREAD

**Discussion by waqasahmed01** [nathanwoulfe/Plumber-2#115] [Q&A] [answered]
Hi 

What configurations are required, if need to change the admin URL /Umbraco to something else? Please let me know.

Thanks

**Comment by nathanwoulfe** [comment]
Did this cause any errors or just stopped working? Anything in the logs?

There shouldn't be any hardcoded references to the Umbraco path, so changing it (in theory) shouldn't have broken anything...

**Reply by waqasahmed01** [reply]
Thank you nathanwoulfe for the quick reply 

All data communication  APIs returning 404. See below for your reference 

![error pic 1](https://user-images.githubusercontent.com/78305293/148895561-aca42a39-84ad-4dc1-83af-c20bc98a5c6a.png)

![pic 2](https://user-images.githubusercontent.com/78305293/148895590-1ad91e5a-daad-40b8-8c12-b1d038e57945.jpg)


Regards

**Reply by nathanwoulfe** [reply]
Interesting... I'll spin it up locally and see what I can find. 

Can you also share the steps you took to change the /Umbraco path, just so I know I follow the exact steps

**Reply by waqasahmed01** [reply]
Here are the web.config settings 
 <add key="Umbraco.Core.ReservedUrls" value="~/secureadmin" />
 <add key="Umbraco.Core.ReservedPaths" value="~/secureadmin" />
 <add key="Umbraco.Core.Path" value="~/secureadmin" />

**Reply by waqasahmed01** [reply]
@nathanwoulfe did you get time to check that.

**Reply by nathanwoulfe** [reply]
No, not yet. Paid work comes first.

**Reply by waqasahmed01** [reply]
thank you, plz take your time and check the issue.

**Comment by nathanwoulfe** [comment]
@waqasahmed01 I've fixed this, will be available in v2.1.1. For now though, and to help with validating the change, v2.1.1-rc will be on Nuget as soon as the build finishes.

Error was due to the Plumber API using attribute routing, which had the /umbraco/ segment hardcoded. I've removed attribute routing, so had to make some updates to the Angular services to pass querystrings rather than routes, so if you notice anything 404'ing, please let me know and I'll fix before releasing 2.1.1.

**Reply by waqasahmed01** [reply]
Thank you @nathanwoulfe 

Is there any workaround for the older version is possible?

**Reply by nathanwoulfe** [reply]
No, unfortunately not - the /umbraco path was used as a routing attribute so is shipped in the compiled library. 

The only other change in the v2.1.1 release is an update to a targets file for netcore, to correctly choose between RoslynCodeTaskFactory and CodeTaskFactory, based on the MS Build version, so isn't a substantial upgrade.

**Reply by waqasahmed01** [reply]
Thank you @nathanwoulfe, waiting for the final release.

**Comment by nathanwoulfe** [comment, marked as answer]
v2.1.1 is available now, fixes this issues 👍
